game/util.cc

```diff
@@ -32,3 +36,112 @@ std::string format(std::chrono::seconds const& unixtime, std::string const& form
 
 	return stream.str();
 }
+
+
+std::string toLower(std::string const& s) {
+	//return boost::algorithm::to_lower_copy(s);
+	auto result = s;
+
+	std::for_each(result.begin(), result.end(), [](auto& c){ c = static_cast<char>(std::tolower(c));});
+
+	return result;
+}
+
+std::string toUpper(std::string const& s) {
```

Comment:
Might be better to add  a locale since toUpper could do something else with weird characters.